deploy/charts/cert-manager/values.secure-defaults.yaml

```diff
@@ -0,0 +1,110 @@
+# Helm chart values which make cert-manager comply with CIS, BSI and NSA
+# security benchmarks.
+# Read https://cert-manager.io/docs/installation/best-practice for more
+# information.
+automountServiceAccountToken: false
+serviceAccount:
+  automountServiceAccountToken: false
+volumes:
+- name: serviceaccount-token
+  projected:
+    defaultMode: 420
+    sources:
+    - serviceAccountToken:
+        expirationSeconds: 3607
+        path: token
+    - configMap:
+        name: kube-root-ca.crt
+        items:
+        - key: ca.crt
+          path: ca.crt
+    - downwardAPI:
+        items:
+        - path: namespace
+          fieldRef:
+            apiVersion: v1
+            fieldPath: metadata.namespace
+volumeMounts:
+- mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+  name: serviceaccount-token
```

Comment:
IMO it's a good idea to add `readOnly: true` here

Reply:
Makes sense.

Reply:
The ServiceAccount admission plugin uses this as the default too:

https://github.com/kubernetes/kubernetes/blob/3992eda8e61725c470fb6141a7fe4e7f9ee31ea5/plugin/pkg/admission/serviceaccount/admission.go#L371-L376